test/e2e_node/resource_usage_test.go

```diff
@@ -142,15 +147,78 @@ func runResourceUsageTest(f *framework.Framework, rc *ResourceCollector, testArg
 		// sleep for an interval here to measure steady data
 		sleepAfterCreatePods = 10 * time.Second
 	)
-	pods := newTestPods(testArg.podsNr, true, framework.GetPauseImageNameForHostArch(), "test_pod")
+	framework.Logf("creating namespaces")
+	var namespaces []*v1.Namespace
+	for i := 0; i < testArg.podsNr; i++ {
```

Comment:
@yujuhong - if we decide that this PR should be merged in some shape, we should probably get rid of that. Since we have explicit secrets, having multiple namespaces is not needed, right?
[And to get service-account-related secrets we would need service account controller and admission plugin, which seems like a waste of time to do.]

WDYT?